config/locales/de.yml

```diff
@@ -196,9 +197,9 @@ de:
           team: "Team"
         bestof_length: "Spielart"
         game_winrule: "Gewinnregel"
-        points_for_win: "Punkte für Gewinn"
-        points_for_draw: "Punkte für Unentschieden"
-        points_for_lose: "Punkte für Niederlage"
+        points_for_win: "Ligapunkte für Gewinn"
```

Comment:
we should definitely talk to the PO again here: This is `event.points_for_win`. Not every event is a league.

Reply:
Rename to Eventpunkte / Eventpoints please. (PO agrees)